lms/envs/production.py

```diff
@@ -1129,3 +1129,7 @@ def get_env_setting(setting):
 ######################### Overriding custom enrollment roles ###############
 
 MANUAL_ENROLLMENT_ROLE_CHOICES = ENV_TOKENS.get('MANUAL_ENROLLMENT_ROLE_CHOICES', MANUAL_ENROLLMENT_ROLE_CHOICES)
+
+########################## limiting dashboard courses ######################
+
+DASHBOARD_COURSE_LIMIT = ENV_TOKENS.get('DASHBOARD_COURSE_LIMIT', 250)
```

Comment:
Default should be none for now, and we can enable it for edx.org via internal overrides first.  This way, the limit isn't on by default and won't require us to rollback the deployment.  If we find that there is a good number for this later, we can put it into the default production settings instead of `None`